internal/cmd/export.go

```diff
@@ -937,8 +948,9 @@ func gatherEvents(ctx context.Context,
 	return nil
 }
 
-// gatherLogs takes a client and buffer to write logs to a buffer
-func gatherPostgresqlLogs(ctx context.Context,
+// gatherAllPostgresLogs take a client and writes logs from primary and replicas
+// to a buffer
+func gatherAllPostgresLogs(ctx context.Context,
```

Comment:
```suggestion
func gatherPostgresLogsAndConfigs(ctx context.Context,
```

To me "gatherAllPostgresLogs" sounds like you're getting all the logs, but does not indicate where those logs are coming from. "gatherPostgresLogsFromAllDatabasePods" might be more indicative of the action you're performing here, but it's a little wordy.

But then what we're really doing here is getting the logs and the confs, so I wanted to make sure we capture that idea in the name.

I'm not sure what the best name here is that balances out "full description" and "not too many words for the sake of legibility".

Reply:
I like `gatherPostgresLogsAndConfigs` - I originally just added `All` because I was getting logs from the primary _and_ the replicas. So it went from gatherPG to gatherAllPG.

I'll refactor to `gatherPostgresLogsAndConfigs`